x-pack/plugins/timelines/public/components/t_grid/integrated/index.tsx

```diff
@@ -303,15 +301,15 @@ const EventsViewerComponent: React.FC<Props> = ({
               data-timeline-id={id}
               data-test-subj={`events-container-loading-${loading}`}
             >
-              <TimelineRefetch
+              {/* <TimelineRefetch
```

Comment:
nit: commented-out code here, [line 312](https://github.com/elastic/kibana/pull/100265/files#diff-5796dde9b49a93dcce15ee9ea2133002123cbb9802033e07ae9db0c9739d08c6R312), and [line 322](https://github.com/elastic/kibana/pull/100265/files#diff-5796dde9b49a93dcce15ee9ea2133002123cbb9802033e07ae9db0c9739d08c6R322)